pkg/sca/sca.go

```diff
@@ -546,6 +546,80 @@ func sonameLibver(soname string) string {
 	return libver
 }
 
+func getShbang(fp fs.File) (string, error) {
+	buf := make([]byte, 80)
+	blen, err := io.ReadFull(fp, buf)
+	if err == io.EOF {
+		return "", nil
+	} else if err == io.ErrUnexpectedEOF {
+		if blen < 2 {
+			return "", nil
+		}
+	} else if err != nil {
+		return "", err
+	}
+
+	if buf[0] != '#' && buf[1] != '!' {
+		return "", nil
+	}
+	toks := strings.Fields(string(buf[2 : blen-2]))
+
+	// if it is not /usr/bin/env, then use it.
```

Comment:
Does it matter if it's `/bin/env` or `/sbin/env` or `/tmp/env` or `/definitely/not/env`? I don't know enough about shebangs or `env` to know if we need to handle those cases.

Reply:
I'm happy to limit it to /usr/bin/env . I don't know that I've ever seen use of 'env' from any other PATH.

I'll update to just do that, and maybe even warn or error if '#!/some/other/path/env'

Reply:
that is done. only /usr/bin/env is treated specially.